tests/test_json_schema.py

```diff
@@ -58,6 +58,7 @@ def test_json_schema():
                             },
                         },
                         "num_trees": {"type": "object"},
+                        "min_bar_length": {"type": "number"},
```

Comment:
Why don't we use the actual schema here? I thought I changed this in an old previous PR :thinking:

Reply:
Ah this change is stuck in #2 :smile:

Reply:
not sure... you can change it in this one if you prefer

Reply:
It depends when you think we will merge #2?

Reply:
ah #2  will likely be abandonned, as we're trying to split it in 2 smaller PRs, maybe create another one with this schema updates?

Reply:
Ah ok ok, I will create a new one then